utils/webhook/github.handler.ts

```diff
@@ -456,3 +432,77 @@ export async function githubWebhookHandler(
         }
     }
 }
+
+async function prepareCommentContent(
+    comment: string,
+    sender?: User,
+    anonymous?: boolean
+) {
+    let modifiedComment = await replaceMentions(comment, "github");
+    modifiedComment = replaceImgTags(modifiedComment);
+
+    if (!anonymous) return modifiedComment;
+
+    return `*GitHub comment from [${sender.login}](https://github.com/${sender.login}):*\n${modifiedComment}`;

```

Comment:
Open to suggestion for different text of the anonymous messages :)

Reply:
Something like this could prioritize the comment content

```javascript
return `>${modifiedComment}\n\n—[${sender.login} on GitHub](${comment.html_url})`
```

Reply:
Good idea!